oneview/resource_storage_system_test.go

```diff
@@ -20,7 +20,7 @@ import (
 )
```

Comment:
same as above

Reply:
StorageSystem Struct got renamed, when updated the govendor dependencies, I got these compilation errors so fixed these.